### ISSUE TITLE
Include HTML test files in packaging

### ISSUE THREAD

**Issue by Juanlu001** [poliastro/poliastro#220]
This is what happens when running the tests from outside the `poliastro` source tree:

```
============================= test session starts ==============================
platform linux -- Python 3.6.2, pytest-3.2.1, py-1.4.34, pluggy-0.4.0
rootdir: /home/juanlu/Personal/poliastro/poliastro, inifile: setup.cfg
collected 99 items                                                              

../home/juanlu/Personal/poliastro/poliastro/src/poliastro/neos/tests/test_dastcom5.py .........
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/neos/tests/test_neos_neows.py ...FFF
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/tests/test_bodies.py ....
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/tests/test_coordinates.py ..
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/tests/test_ephem.py .
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/tests/test_hyper.py ...........
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/tests/test_iod.py .........
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/tests/test_jit.py ..
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/tests/test_maneuver.py ......
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/tests/test_plotting.py ....
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/tests/test_stumpff.py ...
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/tests/test_twobody.py ..
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/tests/test_util.py ....
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/twobody/tests/test_angles.py ....
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/twobody/tests/test_decorators.py ..
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/twobody/tests/test_orbit.py .............
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/twobody/tests/test_propagation.py .......
../home/juanlu/Personal/poliastro/poliastro/src/poliastro/twobody/tests/test_states.py ..........

=================================== FAILURES ===================================
______________________ test_spk_id_from_name_parses_body _______________________

mock_get = <MagicMock name='get' id='139763844725168'>
mock_response = <MagicMock name='Response' id='139763844726176'>

    @mock.patch('poliastro.neos.neows.requests.Response')
    @mock.patch('poliastro.neos.neows.requests.get')
    def test_spk_id_from_name_parses_body(mock_get, mock_response):
>       with open('src/poliastro/neos/tests/table.html', 'r') as demo_html:
E       FileNotFoundError: [Errno 2] No such file or directory: 'src/poliastro/neos/tests/table.html'

/home/juanlu/Personal/poliastro/poliastro/src/poliastro/neos/tests/test_neos_neows.py:60: FileNotFoundError
_____________ test_spk_id_from_name_parses_object_list_and_raises ______________

mock_get = <MagicMock name='get' id='139763844432616'>
mock_response = <MagicMock name='Response' id='139763844432560'>

    @mock.patch('poliastro.neos.neows.requests.Response')
    @mock.patch('poliastro.neos.neows.requests.get')
    def test_spk_id_from_name_parses_object_list_and_raises(mock_get, mock_response):
>       with open('src/poliastro/neos/tests/center.html', 'r') as demo_html:
E       FileNotFoundError: [Errno 2] No such file or directory: 'src/poliastro/neos/tests/center.html'

/home/juanlu/Personal/poliastro/poliastro/src/poliastro/neos/tests/test_neos_neows.py:71: FileNotFoundError
_________________ test_spk_id_from_name_raises_when_not_found __________________

mock_get = <MagicMock name='get' id='139763845095608'>
mock_response = <MagicMock name='Response' id='139763845096112'>

    @mock.patch('poliastro.neos.neows.requests.Response')
    @mock.patch('poliastro.neos.neows.requests.get')
    def test_spk_id_from_name_raises_when_not_found(mock_get, mock_response):
>       with open('src/poliastro/neos/tests/none.html', 'r') as demo_html:
E       FileNotFoundError: [Errno 2] No such file or directory: 'src/poliastro/neos/tests/none.html'

/home/juanlu/Personal/poliastro/poliastro/src/poliastro/neos/tests/test_neos_neows.py:84: FileNotFoundError
===================== 3 failed, 96 passed in 4.49 seconds ======================
```

probably this will involve fixing `MANIFEST.in`.

**Comment by Juanlu001** [comment]
This is still not fixed in #221.

**Comment by Juanlu001** [comment]
Tested and it works. This does not show in the CI though, but it's fixed by #221. Closing.
